src/scripts.js

```diff
@@ -69,13 +62,24 @@ chosenType.value, hotel));
 
 //put login submit button here!!
 
-// roomView.addEventListener("click", (event) => showSelectedRoom(event, hotel))
+chosenType.addEventListener("click", ariaStateChange)
+chosenType.addEventListener("keydown", ariaStateChange)
 
 selectedRoom.addEventListener("click", (event) => bookRoom(event, hotel, customer, bookingsData))
 
 ///Fetch stuff here
 window.addEventListener('load', fetchData);
 
+//Aria
+function ariaStateChange() {
+  let attribute = chosenType.getAttribute("aria-expanded");
```

Comment:
I didn't know that get attribute was a method! Nice

Reply:
Very helpful for screen readers. @hannahhch